stable/wordpress/templates/ingress.yaml

```diff
@@ -1,28 +1,37 @@
-{{- if .Values.ingress.enabled -}}
+{{- if .Values.ingress.enabled }}
+{{- $rootContext := . }}
+{{- range .Values.ingress.hosts }}
```

Comment:
Why do we create multiple Ingress objects when we can create multiple rules in a single Ingress?

Reply:
If there was a desire to have say foo.example.com and insecure-foo.example.com.  Perhaps one (the TLS one) is routed on a externally-reachable ingress controller and the other is not.  And perhaps there are separate ingress annotations for them.  This would not be achievable with just one ingress record afaik

Ingress records are not expensive (storage or otherwise) and I doubt many people will ever have more than one.  But if you did want to, it's there.

That's also why there are separate secret value and annotations array within the `hosts` array.